release-notes-2.7.0.md

```diff
@@ -2,21 +2,23 @@
 ${version-number}
 
 #### New Features
-- [SCMOD-6182](https://portal.digitalsafe.net/browse/SCMOD-6182): Last Updated Time field  
+- [SCMOD-6182](https://portal.digitalsafe.net/browse/SCMOD-6182): Last Updated Time field  
 	When retrieving the status of a job a new `lastUpdateTime` field is returned which provides the last time any activity happened on the job, such as when it was last progressed or completed.
 
-- [SCMOD-6351](https://portal.digitalsafe.net/browse/SCMOD-6351): Prerequisite jobs pre-created declaration  
-	Previously when a job was created with a prerequisite, and that prerequisite job was not found in the system, then it was assumed that the job was yet to be created.  This remains the default behavior, but now an options string can be specified as a suffix with the prerequisite jobs ids, and if one of the options is "`,pc`" (i.e. precreated), then if the job is not found then instead of assuming that it is yet to be created the assumption will be that it was previously created, has completed successfully, and has been deleted. This means that the prerequisite can be effectively ignored.
+- [SCMOD-6351](https://portal.digitalsafe.net/browse/SCMOD-6351): Prerequisite jobs pre-created declaration  
```

Comment:
This ticket is still in progress, this cannot be released until this is resolved.